src/components/FileRepo.js

```diff
@@ -125,9 +125,11 @@ const UploadIdsButton = ({ theme, state, effects, setSQON, ...props }) => (
   </div>
 );
 
+const aggregationsWrapperRef = React.createRef();
```

Comment:
YES!!! React 16.3 rocks!!!!

Reply:
woops! maybe declare this inside `AggregationsWrapper` so we don't have a singleton ref :P